games/m.yaml

```
@@ -554,6 +554,26 @@
   video:
     youtube: cU4T3GSIjxs
 
+- name: mengde
+  originals:
+  - Sangokushi Sōsōden
+  type: remake
+  repo: 'https://github.com/wateret/mengde'
+  development: sporadic
+  status: playable
```

```suggestion
  status: semi-playable
```
from https://github.com/wateret/mengde/blob/master/docs/Introduction.md it seems the game only uses sosoden sprites and basic gameplay without implementing the rest of the game like scenarios